src/DetailsView/components/left-nav/left-nav-link-builder.tsx

```diff
@@ -188,26 +249,33 @@ export class LeftNavLinkBuilder {
         return testLink;
     };
 
+    //requirementIndex is optional so we can create both Assessment-style requirement links
+    //and MediumPass-style Assessment links based on whether it is included or not, respectfully
     private buildRequirementLink(
         deps: AssessmentLinkBuilderDeps,
         test: VisualizationType,
         requirement: Requirement,
         requirementStatus: ManualTestStatus,
-        requirementIndex: number,
         testIndex: number,
         onClick: onTestRequirementClick,
+        requirementIndex?: number,
     ): TestRequirementLeftNavLink {
         const { outcomeTypeSemanticsFromTestStatus, navLinkRenderer } = deps;
         const name = requirement.name;
-        const displayedIndex = `${testIndex}.${requirementIndex}`;
+
+        const displayedIndex = requirementIndex
```

Comment:
Since there are multiple places we are checking for requirement index in this method, I think it makes sense to move it all to one if block with the differential behavior housed there.

Reply:
Alternatively, have another separate method for medium pass instead; it's not doing a whole lot more such that it makes sense to deduplicate it and we do remove the ambiguity if we were to make future changes (while also removing the if check).